### ISSUE TITLE
Preferences: Add component for user preferences

### ISSUE THREAD

**Issue by Fubinator** [surfbirdapp/surfbird#74]
### Description

Right now, there is no way for the user to manage/customize the app. My Idea would be to create a component where the user can manage different settings of the app.

Ideas for such preferences would be:

- Set the language
- Manage Accounts/Clients
- Appereance things
- ...

### Additional Information

The setup for such a component should not be that hard. We need to discuss where a button for the preferences would be placed and if it would be a modal or a component in the content component.
I think we should not begin implementing this before we have implemented a persisted store for vuex (#53), so the preferences do not get lost when restarting the application.

**Comment by pixeldesu** [comment]
I agree with the persisted store being a priority here! (basically this would have been my next step after implementing persistence anyway)

I think a modal component should be sufficient for this!
